### PR TITLE
feat(billing): Uptime DataCategory

### PR DESCRIPTION
**Pull request by jarrettscott**
Adds the `UPTIME` DataCategory

https://www.notion.so/sentry/PRD-Billing-for-Uptime-Monitoring-13d8b10e4b5d80398cc7e52aef0b9306?d=1438b10e4b5d80338873001cbb60e680

### CODE REVIEW

**Review by vbro — APPROVED** [review]
LGTM (billing)
Wait for Relay / Ingest approval before merging

File: CHANGELOG.md [hunk 1]
```diff
@@ -24,6 +24,11 @@
 - Remove metrics summaries. ([#4278](https://github.com/getsentry/relay/pull/4278), [#4279](https://github.com/getsentry/relay/pull/4279), [#4296](https://github.com/getsentry/relay/pull/4296))
 - Use async `redis` for `projectconfig`. ([#4284](https://github.com/getsentry/relay/pull/4284))
 
+## 24.11.2
+
+**Internal**:
+- Adds the UPTIME DataCategory ([#4363](https://github.com/getsentry/relay/pull/4363))
+
```
Comment (Dav1dde) [inline CHANGELOG.md]: This should be in the unreleased section.

**Review by Dav1dde — APPROVED** [review]
Also please add a note to `py/CHANGELOG.md`

File: CHANGELOG.md (hunk 1 above)
Comment (jarrettscott) [reply CHANGELOG.md]: Thank you, updated in [d71f2e2](https://github.com/getsentry/relay/pull/4363/commits/d71f2e26cde54f411b837b3cd8606dd2be251641)

**Pull request merged**